core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -1854,6 +1854,26 @@ secret_configs:
   - cluster: app.ci
     name: crier
     namespace: ci
+- from:
```

Comment:
Is this needed? We can just use the existing secret, since the tool references the kubeconfig names it will use the right ones or not?

Reply:
Which secret do you want me to reuse, `build-farm-credentials`?

The point of this PR is:
when we add a new cluster to our build farm, we do not need to modify the jobs.
So we split the fat `build-farm-credentials` into smaller ones to provide exactly the `kubeconfig`s.

Reply:
That is the kind of information that belongs into the PR description, your reviewer can't read your mind :)

Reply:
Sorry. Forgot to do it.